p2panda-net/src/engine/engine.rs

```diff
@@ -47,7 +47,7 @@ pub enum ToEngineActor<T> {
         topic_id: [u8; 32],
     },
     SyncStart {
-        topic: T,
+        topic: Option<T>,
```

Comment:
With making `topic` optional we can now also send `SyncStart` events for acceptors. This is not very much used, but the symmetry helps to understand what is going on (and might prevent future bugs when we start using the `on_sync_start` method).

Reply:
Nice, I like the symmetry. This also gives us the option to record sync session duration (and other stats) from the "accepting" side in the future.